modules/hitlnext/src/views/full/app/components/Tags.tsx

```diff
@@ -0,0 +1,90 @@
+import { Collapsible, lang } from 'botpress/shared'
+import _ from 'lodash'
+import React, { FC, Fragment, useContext, useState } from 'react'
+import { MenuItem } from '@blueprintjs/core'
+import { MultiSelect } from '@blueprintjs/select'
+
+import { IHandoff } from '../../../../types'
+import { ApiType } from '../../Api'
+import { Context } from '../Store'
+
+interface Props {
+  api: ApiType
+  handoff: IHandoff
+}
+
+const StringMultiSelect = MultiSelect.ofType<string>()
+
+export const Tags: FC<Props> = ({ handoff, api }) => {
+  const { tags, id } = handoff
+  const { state } = useContext(Context)
+
+  const [expanded, setExpanded] = useState(true)
+  const [items, setItems] = useState(_.compact(_.castArray(tags)))
+  const [availableItems, setAvailableItems] = useState(state.config.tags)
+
+  function handleSelect(value, index) {
+    const updated = [...items, value]
+    setItems(updated) // Optimistic update
+    api.updateHandoff(id, { tags: updated }).catch(error => setItems(updated.slice(0, -1)))
+  }
+
+  function handleRemove(value, index) {
+    const updated = _.filter(items, (v, i) => i != index)
+    setItems(updated) // Optimistic update
+    api.updateHandoff(id, { tags: updated }).catch(error => setItems(updated.slice(0, -1)))
+  }
+
+  function renderTag(tag: string) {
+    return tag
+  }
+
+  function renderItem(tag, { modifiers, handleClick }) {
+    if (!modifiers.matchesPredicate) {
+      return null
+    }
+
+    return (
+      <MenuItem
+        active={modifiers.active}
+        disabled={isSelected(tag)}
+        icon={isSelected(tag) ? 'tick' : 'blank'}
+        onClick={handleClick}
+        key={tag}
+        text={tag}
+      />
+    )
+  }
+
+  function filterTag(query: string, item: string) {
+    return item.toLowerCase().indexOf(query.toLowerCase()) >= 0
+  }
+
+  function isSelected(tag: string) {
+    return items.includes(tag)
+  }
+
+  return (
+    <Collapsible
+      opened={expanded}
+      toggleExpand={() => setExpanded(!expanded)}
+      name={lang.tr('module.hitlnext.tags.heading')}
+      ownProps={{ transitionDuration: 10 }}
+    >
+      {!_.isEmpty(state.config.tags) && (
```

Comment:
If we're going to hide tag input when there are no tags configured but keep the collapsible and the divider visible then we need to show something else like an empty state message that says "Configure labels to enable this feature" or something like that. Otherwise we're ending up with an empty tags section  with buggy feeling

Reply:
We can also hide the whole thing (including divider) for now and fix this later along with the two other sections (comments & userProfile) empty states

Reply:
+1 for an empty state explaining how to use the feature